Game_Manual/Basic_Gameplay.md

```diff
@@ -0,0 +1,99 @@
+# Basic Gameplay
+
+There are two objectives of the game: to learn, and to contribute to others' learning.
+
+Focus on these two activities and you will succeed.
+
+### Roles
+
+There are two roles: **Learner** and **Moderator**.
+
+Learners play the game; the Moderator acts as a referee and game master.
+
+### Cycles
+
+The game moves forward through a series of 1-week cycles. Everything happens within a cycle; it is the primary game loop.
+
+Every cycle, learners select learning goals, form projects from them, practice their craft in the context of the project, and finish the cycle by reflecting on both the process and the outcome.
+
+Learners complete cycles on their journey to achieving dignified livelihood.
+
+### Goal Selection
+
+The first stage of a cycle is called **goal selection**.
+
+During this stage, learners vote on goals from the goal library for their particular craft. The goal library for your craft is located in the appropriate craft repo under the [GuildCrafts][guildcrafts] organization.
+
+Each learner chooses their top two goals from the list. It's best to choose goals that are both _relevant_ to the skills you wish to acquire and appropriately _challenging_ given your current skill level.
+
+Use the `/vote` command to vote on goals.
+
+After all learners have voted, the votes are tallied and the most popular goals become **projects** to which learners are then assigned.
+
+### Project Kickoff
+
+A **project** is formed from a goal and is assigned a **team** of learners.
+
+The purpose of a project is for the team to achieve the goal through practicing their craft, learning through collaboration and engagement with relevant and challenging tasks.
+
+#### Artifact and Project Outline
+
+Once projects have been formed, the first thing the team needs to do is to create a container for the **artifact** and write a **project outline** document within it.
+
+The artifact is the output of a project. It is the thing produced by a team to complete a project. All artifacts need a URL and a project outline document. In most cases, the artifact will be a GitHub repository in the account of a learner on the team.
+
+The project outline document defines the specifications and quality criteria for the artifact to meet. To create a project outline, copy the text of the goal into a `README.md` file within your artifact.
+
```

Comment:
Worth adding: "Discuss the specs and criteria with your team, and add them as you see fit to get to a "stretch"